osc-server/src/main/java/org/osc/core/broker/service/tasks/conformance/openstack/deploymentspec/OsDAIConformanceCheckMetaTask.java

```diff
@@ -66,6 +66,9 @@
     DeleteSvaServerTask deleteSvaServerTask;
 
     @Reference
+    DeleteInspectionPortTask deleteInspectionPortTask;
```

Comment:
you might get an NPE for this task as its not part of the create constructor on line 132